setup.py

```diff
@@ -23,7 +23,7 @@ def open_and_read_if_exists(path: str):
 
 setup(
     name='metadata_service',
-    version='2.0.4',
+    version='2.0.5',
```

Comment:
the list of services should not contain `utils` or `data` anymore due to them missing `requirements.txt` files after a refactor on master.